src/platform/tls_schannel.c

```diff
@@ -1076,6 +1115,47 @@ CxPlatTlsAchWorker(
 
 #endif
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+QUIC_STATUS
+CxPlatTlsSetClientCertPolicy(
```

Comment:
`##[warning]src\platform\tls_schannel.c(1123,1): Warning C4142: 'CxPlatTlsSetClientCertPolicy': benign redefinition of type`

Reply:
I don't even understand this error

Reply:
Is the function declared before being used?